### ISSUE TITLE
Create automatic process to upload .jsonld files to a graph instance

### ISSUE THREAD

**Issue by surchs** [neurobagel/bulk_annotations#5]
Given a list of valid Neurobagel .jsonld files that describe annotated datasets (or ids of these datasets that contain also the .jsonld file), I want to have a process that takes a list of these files (or datasets) and uploads them to a graph I have specified, so that the graph ends up as a reflection of the most content of these files.

This could include wiping the graph first and then re-instantiating with the data (i.e. datasets previously in the graph but not part of the upload would be removed).

Related to https://github.com/neurobagel/api/issues/92

**Comment by surchs** [comment]
Coud also fit with https://github.com/neurobagel/api

**Comment by surchs** [comment]
Blocked by #4 (I think)

**Comment by surchs** [comment]
unblocked because #4 is done

**Comment by alyssadai** [comment]
Depending on what you had in mind for this @surchs, I think this issue can maybe be closed since we have a script (currently stored only on our cloud server but should probably be added to the `bagel-cli` repo or elsewhere in the org I think) that takes as input a directory of JSONLD files and adds them all to a specified graph.

If this script also needs to include code to also (or optionally?) *clear* the contents of the specified graph first, that will require some modification. But I think this could be potentially tracked in a separate issue in whatever repo we decide to upload the currently existing script to. 

Thoughts?

**Comment by surchs** [comment]
will be implemented in https://github.com/neurobagel/api/issues/147
